engine-core/src/main/java/io/nosqlbench/engine/core/script/ScenariosExecutor.java

```diff
@@ -113,7 +113,7 @@ public ScenariosResults awaitAllResults(long timeout, long updateInterval) {
         }
         Map<Scenario, ScenarioResult> scenarioResultMap = new LinkedHashMap<>();
         getAsyncResultStatus()
-                .entrySet().forEach(es -> scenarioResultMap.put(es.getKey(), es.getValue().orElseGet(null)));
+                .entrySet().forEach(es -> scenarioResultMap.put(es.getKey(), es.getValue().orElse(null)));
```

Comment:
`orElseGet` takes a `Supplier` whic cant be null... this was probably never hit since we would have seen a NPE otherwise

Reply:
The async view is needed for web services, but I've never seen an NPE or other issue from this code. But yes, the intent was for a value not a supplier.